R/utils.R

```diff
@@ -3,8 +3,6 @@ compact <- function(x) {
   x[!is_empty]
 }
 
-"%||%" <- function(a, b) if (!is.null(a)) a else b
```

Comment:
`{rlang}` exports this operator, and `{rlang}` is a hard dependency of `{devtools}`, so this shouldn't be needed.

Reply:
But atm devtools does not import rlang's entire namespace, so calls to `%||%` don't actually work with this PR. It's regrettable that the tests don't reveal this, but it's true.